### ISSUE TITLE
Summoned Weapons & Enshroud Issues

### ISSUE THREAD

**Issue by CordeliaMist** [Penumbra-Sync/client#53]
**I know initially on first glance people will see enshroud here and go "Oh its known to not work" right away. But allow me to explain.**

After many months of the original report I have made, I have done a lot of research into enshroud and also things like the reaper avatar, and summoned weapons in general.

The commonality between all of them is they are all summoned models in some form or manner, with (mostly) bound animations.

A TMB or PAP file has an effect called "Enshroud" which when called upon summons the reaper enshroud outfit onto the player. If you modify the TMB to call enshroud on other actions besides the ability enshroud, mare will sucessfully call enshroud transformation when you call it via the tmb.

It will not, however, call upon the modded item attached to the enshroud outfit. (if modded).

Consqeuently, you can use a TMB or PAP to call upon summoned weapons, which will call a weapon to your arm and play an animation with it. Such techniques is how reapers avatar is summoned during battle with its oGCD's.

In recent months, other vfx replacement modders have begun getting more reports of parts of their vfx replacement mods not working across mare, and have also received reports of a modded NIN hellfrog summon (called via summon weapon) not displaying via mare as well. If you see the attachment below, you can see what is intended vs what is displayed.
![image](https://github.com/Penumbra-Sync/client/assets/64883480/7926fdac-fb61-4be5-ad64-ea606e457767)
![image](https://github.com/Penumbra-Sync/client/assets/64883480/5faba6c2-714e-4bbb-9d75-9bde4f947a75)
It is clear in the video attached that the animation is being correctly transferred to the player over mare (hence the deformed neck on the frog). Which is leading us to believe that the issue is less exclusive to particuraily enshroud, and more in the scope of anything involving summoned states, failing to store their mesh in mare cache, and only storing the animation. Either this or the fail to store the binding between the two.

Its leading myself and a few others to conclude mare is not properly caching information being brought over on modified pap or tmb's summon weapon model animations / replacements along with enshroud status changes models spesifically.

This issue can be seen in any of the following:
- Any summoned weapon
- Enshroud
- Ninja's frog summon
- Ninja's Bunny summon
- MCH Bowgun

In summary, If the animation is correctly transferring (hence the deformed hellfrog) yet the hellfrog is failing to transfer its modded model, it means that the information mare is failing to transfer is not the declaration of the summoned weapon /animation, but the model file attached to those animations. Which is nice to know considering before all we had to work off was what enshroud could or couldn't do, but this shows the commonality of the issue. The only question now is why this is happening and what needs to be modified to fix this issue.
